python/cudaq/__init__.py

```diff
@@ -183,6 +183,8 @@ def __clearKernelRegistries():
     initKwargs['target'] = sys.argv[sys.argv.index('-target') + 1]
 if '--target' in sys.argv:
     initKwargs['target'] = sys.argv[sys.argv.index('--target') + 1]
+if '--option' in sys.argv:
+    initKwargs['option'] = sys.argv[sys.argv.index('--option') + 1]
```

Comment:
This would be a different command-line interface than C++, no?
Just off the cuff, I wonder if we should choose `--target-option` for both?

Reply:
Agree. It is now `--target-option` for both (C++ and Python CLI).